### ISSUE TITLE
Sign-in code for mobile app?

### ISSUE THREAD

**Issue by Artboy11211** [pi-node/instructions#118]
I have been running the node for 2 months. The sign-in code never stuck when I originally entered it in the mobile app after setting up the node and now I can't find it to add the icon to the mobile app like it appears on the desktop version. Any ideas if/where I can recover the sign-in code from? Small issue but annoying nonetheless.

![IMG_786C75EEC14F-1](https://user-images.githubusercontent.com/83839108/117497121-05836200-af46-11eb-82b6-ad3098713edb.jpeg)

**Comment by nathusius** [comment]
You beat me to it, was about to ask exactly the same question - not a big deal either, just curious really; I don't recall ever seeing any sign-in code in the desktop app!

**Comment by steeveee100** [comment]
Yes it would be good to know if I ever need to reinstall or run the node on another machine. Not simultaneously of course!!

**Comment by remzicirit** [comment]
Same here. I was trying to move node install to my second server which ha smore memory and CPU, it did not work so I moved back to original server. I can see node button on PI Application on the  server I am running but not on the mobile app anymore. Nowhere to link both again. 
![image](https://user-images.githubusercontent.com/83867549/117538353-0a3a2b80-afc3-11eb-8b6e-4efa9294fbff.png)

I do not have same icon on teh mobile application on the phone like you can see mobile application on the server has it

**Comment by nathusius** [comment]
@ihatejam Any ideas about this one?!

**Comment by ihatejam** [comment]
You can't run the node software on a mobile phone so the mobile app should never show the Node icon.

If you need the Node sign-in code from the mobile app, go to its menu on the top left of the front page - "Node" should be near the bottom of the list of menu items.

**Comment by nathusius** [comment]
@ihatejam Thanks, I thought that was the case (can't run node on phone), but the way that menu item is worded (top pic in this thread) says that you are getting a code from the PC (ABCD1234 in the example) to log-in on the phone - that's what confusd me and I guess others too?! I can't remember now, but is that how we originally linked the phone app to the Pi Network on PC?

**Comment by BDL70** [comment]
Has this issue of the sign in code ever been resolved by Pi Network core team members?
Feel like the mobile app should either have the option disconnected/deleted or a fix/tutorial to explain the sign in pop up on mobile devices(in my case iPhone). 
feels like an issue that’s been hanging out in the wind for a loooong time

**Comment by Artboy11211** [comment]
> Has this issue of the sign in code ever been resolved by Pi Network core team members? Feel like the mobile app should either have the option disconnected/deleted or a fix/tutorial to explain the sign in pop up on mobile devices(in my case iPhone). feels like an issue that’s been hanging out in the wind for a loooong time

Crickets. Nothing.
